BackofficeBundle/Resources/public/coffee/page/makeSortable.coffee

```diff
@@ -10,52 +6,17 @@ refreshUl = (ul) ->
       applyClass placeholder
       placeholder.addClass "ui-sortable-placeholder"
       placeholder.html '<div class="preview"></div>'
-    nbrChildren = childs.length
-    direction = if ul.filter(".bo-row").length > 0 then "width" else "height"
-    childs.each ->
-      $(this).css direction, (100 / nbrChildren) + "%"
-      $(this).css (if direction is "height" then "width" else "height"), "auto"
-      return
+    return
   return
 
-makeSortable = (el, duplicate) ->
+
+makeSortable = (el) ->
```

Comment:
You change the function signature, that's ok for me, but I don't see where you're changing the function calling